### DISCUSSION TITLE
search repository through tags

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#38480] [Mobile]
users arent allowed to search repositories through tags, we can search with title, we can search accounts, what if some repository have tags "config, github, discord bot" etc, how are we gonna search those repos through tags?

![Screenshot_2022-11-07-18-26-51-932_com.github.android.jpg](https://user-images.githubusercontent.com/94877880/200316540-7cbee3fe-31bd-42fd-baba-d3e3ee605e28.jpg)

**Comment by david-engelmann** [comment]
[This documentation has a run down of adding topics to a repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics). [Here is the documentation on searching with topics](https://docs.github.com/en/search-github/searching-on-github/searching-for-repositories#search-by-topic)!

**Reply by NitinIsFit** [reply]
it doesnt work

![Screenshot_2022-11-13-06-51-47-609_com github android](https://user-images.githubusercontent.com/94877880/201501036-7164e82b-19cc-4559-aa80-6f49f20afef6.jpg)

**Comment by Dillybar25** [comment]
frpbypass.io
